llvm/utils/update_mc_test_checks.py

```diff
@@ -24,12 +24,24 @@
 OUTPUT_SKIPPED_RE = re.compile(r"(.text)")
 COMMENT = {"asm": "//", "dasm": "#"}
 
+SUBSTITUTIONS = {
+    "%extract-encodings": "sed -n 's/.*encoding://p'",
+}
+
 
 def invoke_tool(exe, check_rc, cmd_args, testline, verbose=False):
-    if isinstance(cmd_args, list):
-        args = [applySubstitutions(a, substitutions) for a in cmd_args]
-    else:
-        args = cmd_args
+    def apply_substitutions(cmd, exe):
```

Comment:
The script already imports UpdateTestChecks/common, can we just use the `apply_substitutions` from there? It looks pretty flexible.

Reply:
Makes sense; this way we could also employ `common.getSubstitutions()`, once needed. Updated.